katalog/gatekeeper/core/deploy.yml

```diff
@@ -4,26 +4,122 @@ kind: Deployment
 metadata:
   labels:
     control-plane: controller-manager
+    gatekeeper.sh/operation: audit
     gatekeeper.sh/system: "yes"
-  name: gatekeeper-controller-manager
+  name: gatekeeper-audit
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      control-plane: audit-controller
+      gatekeeper.sh/operation: audit
+      gatekeeper.sh/system: "yes"
+  template:
+    metadata:
+      annotations:
+        container.seccomp.security.alpha.kubernetes.io/manager: runtime/default
+      labels:
+        control-plane: audit-controller
+        gatekeeper.sh/operation: audit
+        gatekeeper.sh/system: "yes"
+    spec:
+      containers:
+      - args:
+        - --operation=audit
+        - --operation=status
+        - --logtostderr
+        command:
+        - /manager
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        image: openpolicyagent/gatekeeper:v3.1.0
```

Comment:
Can you put the image tag in the kustomization.yaml file?
Something like:

```
image:
  - name: openpolicyagent/gatekeeper
    newTag: v3.1.0
```

Then in the deployment you don't need to specify the tag version